lerna.json

```diff
@@ -8,6 +8,8 @@
     "libraries/botbuilder-core",
     "libraries/botbuilder-dialogs",
     "libraries/botbuilder-testing",
+    "libraries/botbuilder-lg",
+    "libraries/botframework-expressions",
```

Comment:
do we not need a ActivityGenerator? Will that be in lg for now?

Reply:
It will be in as a follow-up PR into lg.